autoinstall/FreeBSD/13/installerconfig

```diff
@@ -78,8 +78,25 @@ done
 
 # Disable ISO repo and enable default repo
 rm -rf /usr/local/etc/pkg/repos/FreeBSD_install_cdrom.conf
+
+# Modify the offical repo to latest
+mkdir -p /usr/local/etc/pkg/repos
+cat >/usr/local/etc/pkg/repos/FreeBSD_latest.conf <<EOF
+FreeBSD_latest: {
+    url: "pkg+http://pkg.FreeBSD.org/\${ABI}/latest",
+    mirror_type: "srv",
+    signature_type: "fingerprints",
+    fingerprints: "/usr/share/keys/pkg",
+    enabled: yes
+}
+EOF
+
 env ASSUME_ALWAYS_YES=YES pkg update -f > /dev/ttyu0
+{% if guest_id.find('64Guest') != -1 %}
```

Comment:
Please do not use guest_id for identifying the OS bitness, we can not trust user's input completely. If it can be get from `uname -m` (for 64bit, its output may be amd64 or x86_64) or environment variable `MACHTYPE` (amd64 or x86_64), we can consider current OS is 64bit FreeBSD. 

Same to line 17.